src/cliAction/startStopDeleteProject/handler_delete.go

```diff
@@ -0,0 +1,64 @@
+package startStopDeleteProject
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/zerops-io/zcli/src/i18n"
+	"github.com/zerops-io/zcli/src/utils"
+	"github.com/zerops-io/zcli/src/zeropsApiProtocol"
+)
+
+func (h *Handler) RunDelete(ctx context.Context, config RunConfig, projectId string, actionType string) error {
+
+	isConfirmed := string(actionType[0]) == "y"
+	if !isConfirmed {
+		// run confirm dialogue
+		answer := askForConfirmation()
+		if !answer {
+			fmt.Println(i18n.CanceledByUser)
+			return nil
+		}
+	}
+
+	deleteProjectResponse, err := h.apiGrpcClient.DeleteProject(ctx, &zeropsApiProtocol.DeleteProjectRequest{
+		Id: projectId,
+	})
+	if err := utils.HandleGrpcApiError(deleteProjectResponse, err); err != nil {
+		return err
+	}
+
+	fmt.Println(i18n.DeleteProjectProcessInit)
+
+	processId := deleteProjectResponse.GetOutput().GetId()
+
+	err = h.checkProcess(ctx, processId)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(i18n.DeleteProcessSuccess)
+
+	return nil
+}
+
+func askForConfirmation() bool {
+	fmt.Print(i18n.ConfirmDelete)
+	var response string
+
+	_, err := fmt.Scan(&response)
+	if err != nil {
+		// fmt.Println(err)
```

Comment:
Do we want to report the error here? :smile:

Reply:
I was not sure, you tell me :D I'd prefer not :D

Reply:
I would also say not. Or you can log it in file using `h.logger`. :smile: but I think it's not important.